helm/values-uat.yaml

```diff
@@ -122,6 +123,7 @@ microservice-chart:
     TRANSACTIONS_REFUND_RETRY_QUEUE_NAME: pagopa-u-weu-ecommerce-transactions-refund-retry-queue
     TRANSACTIONS_REFUND_QUEUE_NAME: pagopa-u-weu-ecommerce-transactions-refund-queue
     TRANSACTIONS_NOTIFICATIONS_RETRY_QUEUE_NAME: pagopa-u-weu-ecommerce-transaction-notifications-retry-queue
+    TRANSACTIONS_NOTIFICATIONS_QUEUE_NAME: pagopa-u-weu-ecommerce-transaction-notifications-retry-queue
```

Comment:
The name of the queue is wrong ?

Reply:
Good catch, my bad. Fixed with https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/56/commits/c8429e16207399bd5615c548c95aa388be1ffb95